activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -758,6 +763,15 @@ def foreign_key_column_for(table_name) # :nodoc:
           super
         end
 
+        def quoted_include_columns_for_index(column_names) # :nodoc:
+          return column_names if column_names.is_a?(Symbol)
```

Comment:
Would you quote `column_names` here?  I wanted every identifier to be quoted. 

`test_include_index` test generates this statement:
```sql
CREATE INDEX "include" ON "ex" ("id")INCLUDE (number)
```

- To be
```sql
CREATE INDEX "include" ON "ex" ("id")INCLUDE ("number")
```

Reply:
✅